src/oop/app/src/Models/Movie.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Create Class - Movie.
+ * Implement the MovieInterface and following methods:
+ * getTitle(), setTitle(), getPoster(), setPoster(), getDescription(), setDescription()
+ * for record and read Movie data.
+ *
+ * Note: Dont forget to create properties for storing Movie data.
+ * Note: Use next namespace for Movie Class - "namespace src\oop\app\src\Models;" (Like in this Interface)
+ *
+ * Note: You need to inject this Class somewhere in the code to get Movie object with film data.
+ * Think about where and how to do it better!!!
+ */
+
+namespace src\oop\app\src\Models;
+
+use src\oop\app\src\Models\MovieInterface;
+
+class Movie implements MovieInterface
+{
+    private $title;
+    private $poster;
+    private $description;
+
+    public function __construct(){
```

Comment:
code style issue { should be in separate line

Reply:
fixed